server/routes/charts.js

```diff
@@ -213,4 +213,30 @@ router
       return res.status(404).json({ message: 'Chart was not duplicated' })
     }
   })
+
+router
+  .route('/api/v1/charts/:chart_id/shared')
+  .patch(ensureAuthenticated, async (req, res) => {
```

Comment:
I think this should be a post to the `share` endpoint or a patch to the `chart` endpoint. Reason being is that we're "creating" a share and not "updating" a share. We are "updating" a chart, though. Does that make sense?

Reply:
Thank you for the new perspective of using a post for this route. 
https://github.com/AMP-SCZ/dpdash/pull/131/commits/789e65512c6e640d9852ea9740fdafc3706c462a